packages/airnode-node/src/types.ts

```diff
@@ -229,6 +229,7 @@ export type ApiCallResponse = RegularApiCallResponse | HttpGatewayApiCallRespons
 export interface RegularApiCallSuccessResponse {
   success: true;
   data: { encodedValue: string; signature: string };
+  gasPriceOverride?: string;
```

Comment:
I spent a lot of time trying to find a way of retaining the requester-specified `_gasPrice` from the where API call response processing and reserved parameter parsing occurs for use where the `gasTarget` created and employed in submitting transactions. Obviously reserved parameters weren't exactly designed / implemented for more than response processing, but adding this property was the simplest solution I came up with.

Reply:
Do you mean something like
```ts
export interface RegularApiCallSuccessResponse {
  success: true;
  data: { encodedValue: string; signature: string };
  reservedParameterOverrides?: {
    _gasPrice?: string;
    _minConfirmations?: string;
  }
}
```

Reply:
Yes.